pkg/generated/restapi/configure_timestamp_server.go

```diff
@@ -71,7 +71,6 @@ func configureAPI(api *operations.TimestampServerAPI) http.Handler {
 
 	api.ServerShutdown = func() {}
 
-	api.AddMiddlewareFor("GET", "/api/v1/timestamp", middleware.NoCache)
 	api.AddMiddlewareFor("GET", "/api/v1/log/timestamp/certchain", cacheForDay)
```

Comment:
I think we need to keep `/api/v1/timestamp`, but update the second one, since the new API path is `/api/v1/timestamp/certchain`

Reply:
Should I update the first one to handle POST requests? I don't see a GET `/api/v1/timestamp` endpoint in the openapi spec.

Reply:
Yep! Since we don’t persist timestamps, it’s only POST